README.md

```diff
@@ -1,5 +1,6 @@
 ![LICENSE](https://img.shields.io/github/license/awnigharbia/filmy)
 [![build](https://github.com/awnigharbia/filmy/actions/workflows/publish.yaml/badge.svg)](https://github.com/awnigharbia/filmy/actions/workflows/publish.yaml)
```

Comment:
```suggestion
[![build](https://github.com/awnigharbia/filmy/actions/workflows/build.yaml/badge.svg)](https://github.com/awnigharbia/filmy/actions/workflows/build.yaml)
```

I think that this should also be renamed since you renamed the file.